### PR TITLE
Kicking ass on Org Routing with Carlos: The Reunion Tour

### PR DESCRIPTION
**Pull request by adamjgrant**
In progress.

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/img/symbol-sprites/svg.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  Setting these SVG properties to "inherit" lets us override them in the
+  <use> tag using CSS
+
+  "For user agents that support Styling with CSS, the generated ‘g’ element
+  carries along with it the "cascaded" property values on the ‘use’ element
+  which result from the CSS cascade"
+
+  http://www.w3.org/TR/SVG/struct.html#UseElement
+*/
+rect, path, circle, g { fill: inherit; stroke: inherit; stroke-width: inherit; }
```
Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/img/symbol-sprites/svg.css:11]: @cvializ Looks like this works already.

screen shot 2015-07-14 at 6 04 13 pm

File: kifi-backend/modules/shoebox/angular/src/common/services/routeService.js [hunk 2]
```diff
@@ -190,6 +190,20 @@ angular.module('kifi')
       },
 
       ////////////////////////////
+      // Org Profile           //
+      ////////////////////////////
+      getOrgProfile: function (handle) {
```
Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/common/services/routeService.js]: @cvializ Did I add this? might need to take it out.

File: kifi-backend/modules/shoebox/angular/img/symbol-sprites/svg.css (hunk 1 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/img/symbol-sprites/svg.css:11]: Yeah I noticed that after you merged that in. We can cut this out then : D

File: kifi-backend/modules/shoebox/angular/src/common/services/routeService.js (hunk 2 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/common/services/routeService.js]: @ajkochanowicz I didn't add it, so you must have : O

**Pull request merged**